src/UI/Components/Diagram/halo.ts

```diff
@@ -113,6 +120,14 @@ const createHalo = (
             filter: "drop-shadow(3px 5px 2px rgb(0 0 0 / 0.4))",
           },
         });
+        const looseElementHighlight = dia.HighlighterView.get(
+          element,
+          "loose_element",
+        );
+
+        if (looseElementHighlight) {
+          V(looseElementHighlight.el).addClass("-hidden");
```

Comment:
What is V ? I see the import, but what does it do?

Reply:
it's tool from jointjs library, [helper for svgs ](https://resources.jointjs.com/docs/jointjs/v4.0/vectorizer.html#V.prototype.addClass)

I remove that, as later on labels branch reminded myself about classList.add/remove (when tried to use it first I saw that it's read-only and switched for that without trying these functions out).